cumulus/parachains/runtimes/assets/asset-hub-westend/src/governance/mod.rs

```diff
@@ -0,0 +1,193 @@
+};
+mod tracks;
+pub use tracks::TracksInfo;
+use xcm_builder::PayOverXcm;
+parameter_types! {
+	pub const VoteLockingPeriod: BlockNumber = prod_or_fast!(7 * RC_DAYS, 1);
+}
+
+mod impls;
+#[cfg(feature = "runtime-benchmarks")]
+use impls::benchmarks::{OpenHrmpChannel, PayWithEnsure};
+
+impl pallet_conviction_voting::Config for Runtime {
+	type WeightInfo = weights::pallet_conviction_voting::WeightInfo<Self>;
+	type RuntimeEvent = RuntimeEvent;
+	type Currency = Balances;
+	type VoteLockingPeriod = VoteLockingPeriod;
+	type MaxVotes = ConstU32<512>; // TODO check with weight
+								// TODO: review
+	type MaxTurnout =
+		frame_support::traits::tokens::currency::ActiveIssuanceOf<Balances, Self::AccountId>;
+	type Polls = Referenda;
+	type BlockNumberProvider = RelaychainDataProvider<Runtime>;
+	type VotingHooks = (); // TODO review
+}
+
+parameter_types! {
+	pub const AlarmInterval: BlockNumber = 1;
+	pub const SubmissionDeposit: Balance = 1 * 3 * CENTS;
+	pub const UndecidingTimeout: BlockNumber = 14 * RC_DAYS;
+}
+
+impl origins::pallet_custom_origins::Config for Runtime {}
+
+parameter_types! {
+	// Fellows pluralistic body.
+	pub const FellowsBodyId: BodyId = BodyId::Technical;
+}
+
+impl pallet_whitelist::Config for Runtime {
+	type WeightInfo = weights::pallet_whitelist::WeightInfo<Self>;
+	type RuntimeCall = RuntimeCall;
+	type RuntimeEvent = RuntimeEvent;
+	type WhitelistOrigin = EitherOfDiverse<
+		EnsureRoot<Self::AccountId>,
+		EnsureXcm<IsVoiceOfBody<Collectives, FellowsBodyId>>,
+	>;
+	type DispatchWhitelistedOrigin = EitherOf<EnsureRoot<Self::AccountId>, WhitelistedCaller>;
+	type Preimages = Preimage;
+}
+
+impl pallet_referenda::Config for Runtime {
+	type WeightInfo = weights::pallet_referenda::WeightInfo<Self>;
+	type RuntimeCall = RuntimeCall;
+	type RuntimeEvent = RuntimeEvent;
+	type Scheduler = Scheduler;
+	type Currency = Balances;
+	type SubmitOrigin = frame_system::EnsureSigned<AccountId>;
+	type CancelOrigin = EitherOf<EnsureRoot<AccountId>, ReferendumCanceller>;
+	type KillOrigin = EitherOf<EnsureRoot<AccountId>, ReferendumKiller>;
+	type Slash = Treasury;
+	type Votes = pallet_conviction_voting::VotesOf<Runtime>;
+	type Tally = pallet_conviction_voting::TallyOf<Runtime>;
+	type SubmissionDeposit = SubmissionDeposit;
+	type MaxQueued = ConstU32<100>;
+	type UndecidingTimeout = UndecidingTimeout;
+	type AlarmInterval = AlarmInterval;
+	type Tracks = TracksInfo;
+	type Preimages = Preimage;
+	type BlockNumberProvider = RelaychainDataProvider<Runtime>;
+}
+
+parameter_types! {
+	pub const SpendPeriod: BlockNumber = 6 * DAYS;
+	pub const Burn: Permill = Permill::from_perthousand(2);
+	pub const TreasuryPalletId: PalletId = PalletId(*b"py/trsry");
+	pub const PayoutSpendPeriod: BlockNumber = 30 * DAYS;
+	// The asset's interior location for the paying account. This is the Treasury
+	// pallet instance (which sits at index 37).
+	pub TreasuryInteriorLocation: InteriorLocation = PalletInstance(37).into();
```

Comment:
@muharem correct? I am seeing `Treasury: pallet_treasury = 94`

Reply:
the whole paymaster is wrongly configured, I will push the change

Reply:
@ggwpez https://github.com/paritytech/polkadot-sdk/pull/7997/commits/090fa91e4415f8b6b45a2c7c0076838af8a3325f